app/ide-desktop/lib/dashboard/src/components/JSONSchemaInput.tsx

```diff
@@ -145,9 +145,7 @@ export default function JSONSchemaInput(props: JSONSchemaInputProps) {
               placeholder="Enter integer"
               onChange={event => {
                 const newValue: number = Math.floor(event.currentTarget.valueAsNumber)
-                if (Number.isFinite(newValue)) {
                   setValue(newValue)
-                }
```

Comment:
This was needed because otherwise I could not have the field empty, so to type `5432` I had to type `05432` and only then delete the `0` afterwards, which was extremely annoying.

Now we can backspace on the `0` and type the correct port name. Empty field still has the red border as 'invalid input'.